app/assets/javascripts/Components/test_script_result_table.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {render} from 'react-dom';
+import ReactTable from 'react-table';
+import _ from 'lodash';
```

Comment:
`lodash` is not an existing dependency, nor should you be adding a new dependency. Look for another way to do what you want to do.